activerecord/lib/active_record/encryption/encryptable_record.rb

```diff
@@ -82,7 +82,7 @@ def global_previous_schemes_for(scheme)
           def encrypt_attribute(name, attribute_scheme)
             encrypted_attributes << name.to_sym
 
-            attribute name do |cast_type|
+            attribute name, default: -> { columns_hash[name.to_s]&.default } do |cast_type|
```

Comment:
Above this we're calling `to_sym` on `name`. Do we need the `to_s` here then? Or is it that `name` can be either a String or Symbol and we don't know at this point?

Reply:
Yes, that `name` comes from the `.encrypts` declaration, and it can be either a symbol or a string. Symbols is definitely the canonical way, but strings can be interesting when declaring encrypted attributes from an array of them (e.g: `encrypts *ATTRIBUTES_TO_ENCRYPT`).